libnmstate/nm/bridge.py

```diff
@@ -68,6 +80,31 @@ def _set_bridge_properties(bridge_setting, options):
             bridge_setting.props.multicast_snooping = val
         elif key == LB.STP_SUBTREE:
             _set_bridge_stp_properties(bridge_setting, val)
+        elif nm_version_bigger_or_equal_to("1.25.2"):
+            if key == LB.Options.GROUP_ADDR:
+                bridge_setting.props.group_address = val.lower()
+            elif key == LB.Options.HASH_MAX:
+                bridge_setting.props.multicast_hash_max = val
+            elif key == LB.Options.MULTICAST_LAST_MEMBER_COUNT:
+                bridge_setting.props.multicast_last_member_count = val
+            elif key == LB.Options.MULTICAST_LAST_MEMBER_INTERVAL:
+                bridge_setting.props.multicast_last_member_interval = val
+            elif key == LB.Options.MULTICAST_MEMBERSHIP_INTERVAL:
+                bridge_setting.props.multicast_membership_interval = val
+            elif key == LB.Options.MULTICAST_QUERIER:
+                bridge_setting.props.multicast_querier = val
+            elif key == LB.Options.MULTICAST_QUERIER_INTERVAL:
+                bridge_setting.props.multicast_querier_interval = val
+            elif key == LB.Options.MULTICAST_QUERY_USE_IFADDR:
+                bridge_setting.props.multicast_query_use_ifaddr = val
+            elif key == LB.Options.MULTICAST_QUERY_INTERVAL:
+                bridge_setting.props.multicast_query_interval = val
+            elif key == LB.Options.MULTICAST_QUERY_RESPONSE_INTERVAL:
+                bridge_setting.props.multicast_query_response_interval = val
+            elif key == LB.Options.MULTICAST_STARTUP_QUERY_COUNT:
+                bridge_setting.props.multicast_startup_query_count = val
+            elif key == LB.Options.MULTICAST_STARTUP_QUERY_INTERVAL:
+                bridge_setting.props.multicast_startup_query_interval = val
```

Comment:
In order to avoid big `if/elif` statements, this can be solved with a dict containing `{nmstate_key:prop}` and a for loop. What do you think? Thanks!

Reply:
Changed to dict